supervisor.py

```diff
@@ -138,7 +90,7 @@ def wait(self):
 
     # Try to pull
     print("We're going to update your instance of redbot now.")
-    updates = subprocess.Popen(['git', 'pull'])
+    updates = subprocess.run(['git', 'pull'])
     if updates.wait() != 0:
         print("Error updating Redbot. Please investigate.")
         sys.exit(0)
```

Comment:
could go with

```python
if subprocess.run(['git', 'pull']).returncode:   # since .run does .wait automatically
    print("Error updating Redbot. Please investigate.")
    sys.exit(1)    # maybe not exit code of zero if its an error?
```